scripts/gen-deps-list

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env node
```

Comment:
Now that I think about it, I think we should add the `.js` extension to this file to make sure ESLint and Prettier correctly handle it.